hadoop-plugin-test/expectedJobs/basicFlow/basicFlow.flow

```diff
@@ -0,0 +1,25 @@
+config:
+  flow-level-parameter: value
+nodes:
+- name: basicFlow
```

Comment:
Why do you need to create two noop jobs: basicFlow and shellEnd? Will one be enough?

Reply:
Discussed offline. You are right! I removed shellEnd jobs.